x-pack/plugins/security_solution/server/endpoint/services/actions/action_list.ts

```diff
@@ -290,6 +292,8 @@ const getActionDetailsList = async ({
       agentState,
       isExpired,
       status,
+      // 8.8 onwards, show outputs only if action is completed for matching requested action ids
+      outputs: isCompleted && withOutputs && withOutputs.includes(action.id) ? outputs : undefined,
```

Comment:
I think `isComplete` should be removed here. We should return `outputs` regardless if the action is complete or not.

Reply:
09e97ffb6f3c163eae4c0036b26ed9d6c131d7b1